src/routes/(admin)/tests/test-[type]/+page.svelte

```
@@ -36,6 +36,20 @@
 
 	let currentScreen: number = $state(typeOfScreen.main);
 
+	let filteredTags: String[] = $state([]);
```

ESLint

[error] Prefer using the primitive `string` as a type name, rather than the upper-cased `String`.

(@typescript-eslint/no-wrapper-object-types)